src/api/users.ts

```diff
@@ -0,0 +1,37 @@
+import express from "express";
+import { PrismaClient } from "@prisma/client";
+
+const prisma = new PrismaClient();
+
+const usersRouter = express.Router();
+
+// GET /api/users
+
+usersRouter.get("/", async (req, res, next): Promise<void> => {
```

Comment:
You may want to add a TODO here of some sort to restrict or delete this endpoint. eg. maybe only admins can see all users, or maybe only logged in users?

Reply:
I'd be fine deleting it.